index.bs

```diff
@@ -454,7 +460,7 @@ spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
     portals-nested.html
   </wpt>
 
-  Whenever a <{portal}> element |element| is [=removing steps|removed=], run the following steps:
+  Whenever a <{portal}> element |element| [=becomes browsing-context disconnected=], run the following steps:
```

Comment:
These will now run in the BC gets discarded (e.g. if you have a Document with a portal, and you .close() its containing Window).

Reply:
That sounds reasonable — the guest contents should be discarded in that case.